Python 2.7.1 (r271:86832, Nov 27 2010, 18:30:46) [MSC v.1500 32 bit (Intel)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> ================================ RESTART ================================
>>> 
No handlers could be found for logger "base.gui.Window"
browser
True
lol
False
qt4plugin
False
search
False
Traceback (most recent call last):
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\model\LDAPTreeItemModel.py", line 132, in rowCount
    self.populateItem(parent)
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\model\LDAPTreeItemModel.py", line 183, in populateItem
    list = parentItem.fetchChildList()
AttributeError: 'NoneType' object has no attribute 'fetchChildList'
Traceback (most recent call last):
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\model\LDAPTreeItemModel.py", line 93, in index
    childItem = parentItem.child(row)
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\item\AbstractLDAPTreeItem.py", line 57, in child
    return self.childItems[row]
IndexError: list index out of range
browser
True
lol
False
qt4plugin
False
search
False
Traceback (most recent call last):
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\model\LDAPTreeItemModel.py", line 132, in rowCount
    self.populateItem(parent)
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\model\LDAPTreeItemModel.py", line 183, in populateItem
    list = parentItem.fetchChildList()
AttributeError: 'NoneType' object has no attribute 'fetchChildList'
Traceback (most recent call last):
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\model\LDAPTreeItemModel.py", line 93, in index
    childItem = parentItem.child(row)
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\plugins\browser_plugin\item\AbstractLDAPTreeItem.py", line 57, in child
    return self.childItems[row]
IndexError: list index out of range

Traceback (most recent call last):
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\lumaWithOptions.py", line 268, in <module>
    main(sys.argv)
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\lumaWithOptions.py", line 234, in main
    startApplication(argv, opt.verbose, clear, dirs)
  File "C:\Documents and Settings\Eier\Mine dokumenter\Einar\devel\luma\luma\lumaWithOptions.py", line 140, in startApplication
    sys.exit(app.exec_())
SystemExit: 0
>>> 
